app/views/shoes.erb

```diff
@@ -0,0 +1,24 @@
+<div class="content">
+  <div id="shoes">
+  <% Shoe.all.each_slice(2) do |slice| %>
+  <table id="shoe_table">
+    <tr>
+      <td>
+        <% col = 0 %>
```

Comment:
This has a lot of duplicate code. Your use of the `col` variable illustrated this. Use an ERB partial :)